stable/nats/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
```

Comment:
why do you want this NetworkPolicy?

Reply:
It's a good feature to have. You can specify what pods have privileges to access your NATS service.

Reply:
👍